nmt_test.go

```diff
@@ -596,12 +596,17 @@ func TestNamespacedMerkleTree_calculateAbsenceIndex_Panic(t *testing.T) {
 		nID      namespace.ID
 		pushData []namespaceDataPair
 	}{
+		{"empty tree", []byte{0, 0}, []namespaceDataPair{}},
 		{"((0,0) == nID < minNID == (0,1))", []byte{0, 0}, generateLeafData(nidLen, 1, 3, []byte{})},
 		{"((0,3) == nID > maxNID == (0,2))", []byte{0, 3}, generateLeafData(nidLen, 1, 3, []byte{})},
```

Comment:
We could also modify the test names of these two to be consistent with the first one e.g., "non-empty tree with 3 leaves: ((0,0) == nID < minNID == (0,1))"